packages/kbn-expandable-flyout/src/types.ts

```diff
@@ -8,6 +8,11 @@
 
 import React from 'react';
 
+export interface PanelPath {
+  tab: string;
+  subTab?: string;
```

Comment:
I know these are obvious, but could you add documentation to these two? Only reason is because this is a reusable package, so having it fully documented is nice